explorer/pages/explore/[slug]/[...uuid].tsx

```diff
@@ -13,7 +13,7 @@ import { Project } from "../../../app/views/Project/project";
 
 interface PageUrl extends ParsedUrlQuery {
   slug: string;
-  uuid: string;
+  uuid: string[];
```

Comment:
Can you tell me why uuid is now a string?

Reply:
This is because now we're using dynamic a route to catch all routes under /explore/[slug]/, since we introduced tabs under this path.